spec/traits.dd

```diff
@@ -939,6 +940,9 @@ $(H2 $(LNAME2 specialkeywords, Special Keywords))
     $(P $(CODE __FILE__) and $(CODE __LINE__) expand to the source
```

Comment:
I figured the documentation was vague because the behavior was meant to be undefined.  It lets the compiler do whatever it wants, which is ok IMO.  That's part of the reason why I thought adding `__FILE_FULL_PATH__` was a good idea.  Let `__FILE__` remain vague so the compiler can decide what's best to do, but require that `__FILE_FULL_PATH__` always be absolute in case the application needs a guaranteed behavior.